packages/react-ui/src/localization/en.js

```diff
@@ -0,0 +1,7 @@
+import c4r from './en/c4r.js';
+
+const locales = {
+  c4r
```

Comment:
Probably at this point we can just put the c4r.js content here. And if we eventually need to split this into several files, we could have one file per package (so one file per `widgets`, other for `ui`, other fore `basemaps`, etc)

Reply:
Ok 👍 , that has all the sense to me